core/broadleaf-framework/src/main/java/org/broadleafcommerce/core/util/service/ResourcePurgeServiceImpl.java

```diff
@@ -102,6 +110,15 @@
     @Resource(name = "blCustomerService")
     protected CustomerService customerService;
 
+    @Resource(name = "blcDeleteStatementGenerator")
```

Comment:
As mention above, change from "blcDeleteStatementGenerator" to "blDeleteStatementGenerator"